docs/template_plugin/src/CMakeLists.txt

```diff
@@ -30,7 +31,6 @@ target_link_libraries(${TARGET_NAME} PRIVATE
 
 set_target_properties(${TARGET_NAME} PROPERTIES INTERPROCEDURAL_OPTIMIZATION_RELEASE ${ENABLE_LTO})
 
-# ATTENTION: uncomment to register a plugin in the plugins.xml file
-# ie_register_plugins(MAIN_TARGET ${TARGET_NAME}
-#                     POSSIBLE_PLUGINS ${TARGET_NAME})
+# ATTENTION: uncomment to register plugins in the plugins.xml file
+# ov_register_plugins(MAIN_TARGET ${TARGET_NAME})
```

Comment:
maybe we need to have cmake variable `ENABLE_TEMPLATE_REGIST` ? instead of manual fixing the source code each time

Reply:
This would be super useful from the testing perspective. To run the tests with the TEMPLATE plugin right now we have to modify this cmake file before starting the build. If possible please add a variable controlling the registration of this plugin.